platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -605,11 +615,16 @@ public class SwirldsPlatform implements Platform {
                 initialState.getState(),
                 appVersion);
 
+        final InterruptableConsumer<ReservedSignedState> newSignedStateFromTransactionsConsumer = rs -> {
+            latestCompleteState.newIncompleteState(rs.getAndReserve("latestCompleteState.newIncompleteState"));
+            stateManagementComponent.newSignedStateFromTransactions(rs);
+        };
```

Comment:
Also something that seems to be a good candidate for the wiring framework.

Reply:
in this particular instance, once the state hashing becomes a component, this will be part of the wiring. this will only be after the SSM becomes a component